src/graphql/fragments/loanChannelFields.graphql

```diff
@@ -3,7 +3,6 @@
 fragment loanChannelFields on LoanChannel {
 	id
 	name
-	description
```

Comment:
I suspect this may come back at some point, but I guess there's no harm in not pulling it until we need it.